docs/CONFIG.md

```diff
@@ -3488,6 +3519,12 @@ LimitDefault = 500_000 # Default
 LimitDefault sets default gas limit for outgoing transactions. This should not need to be changed in most cases.
 Some job types, such as Keeper jobs, might set their own gas limit unrelated to this value.
 
+### LimitMax<a id='EVM-GasEstimator-LimitMax'></a>
+```toml
+LimitMax = 5_000_000 # Default
```

Comment:
I see the actual default set everywhere is 50_000_000(50m), not 5m.

Reply:
Ah right, we don't have validation of defaults for new config yet. Fixed and added link to the story on the todo comment.